### PR TITLE
feat: add issue form for github profile review

### PR DESCRIPTION
**Pull request by schmelto**
add issue form for github profile review

### CODE REVIEW

**Review by krishdevdb — DISMISSED** [review]

**Comment by krishdevdb** [comment]
How do I become a machine like you? You review all of the projects/profiles/resumes... open so many prs... You are a true machine...

**Review by vinzvinci — APPROVED** [review]
LGTM 👍 This form will be helpful, cause some folks linking other projects on gh-profile review.

**Review by Cahllagerfeld — APPROVED** [review]
Looks fine I think 👍

**Comment by schmelto** [comment]
> LGTM 👍 This form will be helpful, cause some folks linking other projects on gh-profile review.

Yes. In future I think we should convert allissue templates to forms. It would be really cool of forms have an of implementation like when I add a github link add the laben profile Review if it's an website linke make resume etc. 🙈

**Comment by schmelto** [comment]
> Looks fine I think 👍 

I think :D

**Review by deephunt3r — DISMISSED** [review]
LGTM 🚀

File: .github/ISSUE_TEMPLATE/gh-profile-review.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    attributes:
+      label: GitHub Profile Link
+      placeholder: https://github.com/<your-username>
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        You can also join the Discord community [here](http://discord.eddiehub.org)
+        Feel free to check out other cool repositories of EddieHub Community [here](https://github.com/EddieHubCommunity)
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/gh-profile-review.yml:17]:
```suggestion
        Feel free to check out other cool repositories of the EddieHub Community [here](https://github.com/EddieHubCommunity)
```

**Review by Panquesito7 — COMMENTED** [review]
LGTM, apart from the comment below. 👍

**Review by Panquesito7 — APPROVED** [review]
Thank you; this PR can be merged. 👍

**Review by Vyvy-vi — APPROVED** [review]
LGTM!

**Comment by Vyvy-vi** [comment]
Thanks for this contribution
Yaay, new issue-forms :)

**Pull request merged**